dao/src/main/java/com/fitgoal/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,73 @@
+package com.fitgoal.dao.impl;
+
+import com.fitgoal.dao.UserDao;
+import com.fitgoal.dao.domain.UserDto;
+import com.fitgoal.dao.mybatis.config.MyBatisConfig;
+import com.fitgoal.dao.mybatis.mappers.UserMapper;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionException;
+
+import java.util.Optional;
+
+
+public class UserDaoImpl implements UserDao {
+
+    public Optional<UserDto> getById(Long id) {
+        Optional<UserDto> user = Optional.empty();
```

Comment:
Redundant instance. Return fetched result in `try` and `Optional.empty()` in `catch`

Reply:
Done